content/foundations/physical_properties/induced_polarization_physical_properties_duplicate.rst

```diff
@@ -3,6 +3,364 @@
+| graphite              | 11.2                     |  
++-----------------------+--------------------------+
+| chalcopyrite          | 9.4                      |  
++-----------------------+--------------------------+
+| bornite            	| 6.3                      |  
++-----------------------+--------------------------+
+| galena                | 3.7                      |  
++-----------------------+--------------------------+
+| magnetite             | 2.2                      |  
++-----------------------+--------------------------+
+| malachite             | 0.2                      |  
++-----------------------+--------------------------+
+| hematite              | 0.0                      |  
++-----------------------+--------------------------+
+
+
+
+
+
+
+Factors Impacting Chargeability
+===============================
+
+
+**Sulphide Mineralization:**
+
+As we discussed earlier, electrode polarization occurs when the pore path is blocked by metallic particles.
+A major source of these metallic particles is sulphide mineralization.
+As the abundance of sulphide minerals within a rock increases, so does the electrode polarization.
+Therefore, highly mineralized rock tend to be very chargeable.
+
+
+**Clays:**
+
+ .. figure:: ./images/images_duplicates/memb_pol_2nd_type.gif
+	:align: right
+	:scale: 100	
+
+Clays have a tendancy to partially block the path which ions take through the rock's pore water.
+Upon application of an electric potential, positive charge carriers pass easily, while negative carriers accumulate.
+This results in an "ion-selective" membrane polarization.
+Clays represent a dominant source of induced polarization in unmineralized sedimentary rocks.
+
+A surplus of both cations and anions occurs at one end of the membrane, while a deficiency occurs at the other end.
+The reduction of mobility is most obvious at frequencies slower than the diffusion time of ions between adjacent membrane zones; i.e. slower than around 0.1 Hz. Conductivity increases at higher frequencies.
+
+
+**Pore-Water Salinity:**
+
+
+The induced polarization within a rock depends on having a mechanism for accumulating ionic charges.
+It also depends on the salinity of the pore water; i.e. the concentration of ions within the pore water.
+As the pore-water salinity increases, so does the capacity of the rock to support a build-up of ionic charges.
+This results in an increases chargeability for the rock.
+
+
+**Tortuosity:**
+
+Tortuosity defines the connectivity and complexity of a rock's pore-space network.
+As the tortuosity of the rock's pore-space increases, it becomes more difficult for ionic charges to move through the rock.
+As a result, and increases abundance of ionic charges will accumulate within the rock when it is subjected to an electric field.
+Thus, the chargeability of a rock increases and its tortuosity increases.
+
+
+
+
+
+
+xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
```

Comment:
@dccowan  is material below this old? could you please comment it out or remove it if so?